connectors/vmware/src/centreon/vmware/cisTags.pm

```diff
@@ -214,8 +215,21 @@ sub tagsByResource {
             );
             return $code if ($code);
 
+            if (!defined($categories->{$tag_detail->{value}->{category_id}})) {
+                my ($code, $category_detail) = $self->request(
+                    method => 'GET',
+                    endpoint => '/rest/com/vmware/cis/tagging/category/id:' . $tag_detail->{value}->{category_id}
+                );
+                return $code if ($code);
+                $categories->{$tag_detail->{value}->{category_id}} = $category_detail->{value}->{name};
```

Comment:
```suggestion
                $categories->{$tag_detail->{value}->{category_id}} = $category_detail->{value}->{name} unless ($code);
```

Reply:
Can you test this and confirm us it's better not to return here if the new request fails?